release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -55,6 +55,10 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-4-17-installation-and-update_{context}"]
 === Installation and update
 
+[id="ocp-4-17-installation-and-update-azure-capi_{context}"]
+==== Cluster API replaces Terraform for {azure-first} installations
+In {product-title} 4.17, the installation program uses Cluster API instead of Terraform to provision cluster infrastructure during installations on {azure-short}.
```

Comment:
After switching to use Cluster API, more permissions are required if user uses service principal with minimal permissions to install cluster, it's better to highlight here.

Missed permissions are tacked via https://issues.redhat.com/browse/OCPBUGS-37701.

Reply:
Thank you @jinyunma I have updated this PR with a note about the permissions, and added the permissions to the main documentation in a separate PR: https://github.com/openshift/openshift-docs/pull/82095 Please review, thanks

Reply:
Thanks for the update, the change looks good to me.